ReactWindows/ReactNative/Views/Image/ColorFilterPostprocessor.cs

```diff
@@ -0,0 +1,69 @@
+﻿using ImagePipeline.Request;
+using Windows.Graphics.Imaging;
+
+namespace ReactNative.Views.Image
+{
+    /// <summary>
+    /// A postprocessor that applies tint and background color.
+    /// </summary>
+    public class ColorFilterPostprocessor : BasePostprocessor
+    {
+        private uint? _tintColor;
+        private uint? _backgroundColor;
+
+        /// <summary>
+        /// Instantiates the <see cref="ColorFilterPostprocessor"/>.
+        /// </summary>
+        public ColorFilterPostprocessor(
+            uint? tintColor,
+            uint? backgroundColor)
+        {
+            _tintColor = tintColor;
+            _backgroundColor = backgroundColor;
+        }
+
+        /// <summary>
+        /// Process the bitmap pixel data.
+        /// </summary>
+        /// <param name="data">The bitmap pixel data.</param>
+        /// <param name="width">
+        /// The width of the new software bitmap, in pixels.
+        /// </param>
+        /// <param name="height">
+        /// The height of the new software bitmap, in pixels.
+        /// </param>
+        /// <param name="format">
+        /// The pixel format of the new software bitmap.
+        /// </param>
+        /// <param name="alpha">
+        /// The alpha mode of the new software bitmap.
+        /// </param>
+        public override void Process(
```

Comment:
I'm not certain this background color filter is quite what we need. I don't think it deals with modifying partially transparent pixels.

Since the composition approaches work very well, maybe we should put this "patch" in a separate PR, so we can revert / decide on them separately if necessary.